recipes/effcee/all/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

The CI complains that you need to use Cmake 3.1 because of CMAKE_CXX_STANDARD